app/controllers/posts_controller.rb

```diff
@@ -1,18 +1,20 @@
 class PostsController < ApplicationController
   before_action :logged_in_user, only: [:create]
 
-  def show
-    @post = Post.find(params[:id])
-  end
-
   def new
     @post = Post.new
   end
 
+  def show
+    @post = Post.find(params[:id])
+    @comments = @post.comments.order('created_at ASC')
+    @comment = Comment.new
```

Comment:
I wasn't sure whether comments had a `post_id` column on them or whether comments and posts were tied together through the commentable polymorphic association.  I'm also not clear how `@comment` is being used in the view. I was just suggesting the automatic setting of `post_id` if that were a thing that you'd want to pre-populate where `@comment` is used.  Hopefully my rambling makes sense.